### ISSUE TITLE
Secret Weapons of the Luftwaffe (1993)

### ISSUE THREAD

**Issue by Burrito78** [TotalDOSCollection/TDCbugs#17]
Files:
Secret Weapons of the Luftwaffe 2.1 to 2.2 update (1993)(LucasArts Entertainment Company LLC) [Action, Simulation, Addon].zip
Secret Weapons of the Luftwaffe v1.0 (1991)(LucasArts Entertainment Company LLC) [Action, Simulation, Strategy].zip
Secret Weapons of the Luftwaffe v1.0 [a1] (1991)(LucasArts Entertainment Company LLC) [Action, Simulation, Strategy].zip

Issues:
First file is a fan patch and not an official release.
Last two files are not original but cracked to remove cp.

![Bildschirmfoto 2024-06-25 um 12 29 22](https://github.com/TotalDOSCollection/TDCbugs/assets/61995128/aa9490da-9e9c-4715-aaf1-fbcb46ddd8e9)

**Comment by TotalDOSCollection** [comment]
ok, I think best course on #1 is that I just create an unzip-n-run playable version of v2.2 from air combat classics (where these files were sourced from) and that way we'll still have the files but not in this weird "unofficially released, but actual released files" state.

#2 and 3 can likely get replaced with a proper pull from the kfx/IMA set which we have as known good dumps.

**Comment by TotalDOSCollection** [comment]
`
Deleted:Secret Weapons of the Luftwaffe 2.1 to 2.2 update (1993)(LucasArts Entertainment Company LLC) [Action, Simulation, Addon].zip-->unofficial and duplicated on air classics
Added:Secret Weapons of the Luftwaffe v2.2 (1994)(LucasArts Entertainment Company LLC) [Action, Simulation, Strategy].zip
Replaced:Secret Weapons of the Luftwaffe v1.0 [a1] (1991)(LucasArts Entertainment Company LLC) [Action, Simulation, Strategy].zip-->Secret Weapons of the Luftwaffe v1.00 (1991)(LucasArts Entertainment Company LLC) [Action, Simulation, Strategy][!].zip
Deleted:Secret Weapons of the Luftwaffe v1.0 (1991)(LucasArts Entertainment Company LLC) [Action, Simulation, Strategy].zip-->diff crack, not needed after [!] version added

`

Indeed both copies for #2 here were replacements for a copy taken directly off the known good kfx/ima set we have.
The two are cracked slightly differently- I picked the one from [a1] but documented both in the wiki.
In that [a1] install.exe is taken from v1.01 and install2.exe is from v1.02 (or vice-versa) so no need for that one to exist in TDC as we already have both of those versions.
